kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/CardThumbnail.vue

```diff
@@ -47,18 +47,8 @@
     },
     computed: {
       displayDurationChip() {
-        if (this.hideDuration) {
-          return false;
-        } else if (this.isMobile) {
-          return false;
-        }
-        return true;
-      },
-      checkIfExercise() {
-        if (this.contentNode.kind === 'exercise') {
-          return true;
-        }
-        return false;
+        // Hide if hideDuration or isMobile is true, display if contentNode kind is exercise
+        return this.hideDuration || this.isMobile ? false : this.contentNode.kind === 'exercise';
```

Comment:
This is more cleaner and understandable!

Reply:
Maybe even the below for more conciseness?(Not a blocker though, l'll leave you to choose 🙂  )
 `return !(this.hideDuration || this.isMobile) && this.contentNode.kind === 'exercise';`